pyiron_base/job/util.py

```diff
@@ -156,7 +156,7 @@ def _is_valid_job_name(job_name):
     """
     if not job_name.isidentifier():
         raise ValueError(
-            'Invalid name for a PyIron object, must be letters, digits (not as first character) and "_" only'
+            f'Invalid name for a PyIron object, must be letters, digits (not as first character) and "_" only, not {job_name}'
```

Comment:
Update pyiron spelling as well:
```suggestion
            f'Invalid name for a pyiron object, must be letters, digits (not as first character) and "_" only, not {job_name}'
```